includes/controls/groups/border.php

```diff
@@ -73,12 +73,14 @@ protected function init_fields() {
 			],
 		];
 
+		// To do: remove after we update all widgets border-width properties to logical.
+		$border_physical_properties = 'border-width: {{TOP}}{{UNIT}} {{RIGHT}}{{UNIT}} {{BOTTOM}}{{UNIT}} {{LEFT}}{{UNIT}};';
 		$fields['width'] = [
 			'label' => esc_html_x( 'Width', 'Border Control', 'elementor' ),
 			'type' => Controls_Manager::DIMENSIONS,
 			'size_units' => [ 'px', 'em', 'rem', 'vw', 'custom' ],
 			'selectors' => [
-				'{{SELECTOR}}' => 'border-width: {{TOP}}{{UNIT}} {{RIGHT}}{{UNIT}} {{BOTTOM}}{{UNIT}} {{LEFT}}{{UNIT}};',
+				'{{SELECTOR}}' => "$border_physical_properties border-block-start-width: {{TOP}}{{UNIT}}; border-inline-start-width {{RIGHT}}{{UNIT}}; border-block-end-width {{BOTTOM}}{{UNIT}}; --border-inline-start-width {{LEFT}}{{UNIT}};",
```

Comment:
I believe we can revert the changes, I don't see a problem with the original code.

Reply:
@hein-obox  reverting, heads up this is tied to logical border props in Mega Menu around line 1835 & the same in Mega Menu html

Reply:
I would like to revert this change.